src/sentry/suspect_resolutions/metric_correlation.py

```diff
@@ -0,0 +1,45 @@
+from datetime import timedelta
+
+from sentry import tsdb
+from sentry.models import Group
+
+
+def is_issue_error_rate_correlated(resolved_issue_id: int, issue2_id: int) -> bool:
+    resolved_issue = Group.objects.filter(id=resolved_issue_id)
+
+    if len(resolved_issue) != 1:
+        return False
+
+    resolution_time = resolved_issue.values_list("resolved_at", flat=True)[0]
+
+    start_time = resolution_time - timedelta(hours=1)
+    end_time = resolution_time + timedelta(hours=1)
```

Comment:
This looks like its capturing 2 hours of issue-error data - is this what we want?

Reply:
Yeah, the idea is to compare the error rate an hour before the issue was resolved to the error rate an hour after. We could change it to a 30-minute window if there are concerns with performance, but I decided to go with the 1-hour timeframe so that there’s more data used in the calculation of the Pearson correlation coefficient to increase accuracy.